docs/src/api/params.md

```diff
@@ -356,8 +356,14 @@ Emulates consistent window screen size available inside web page via `window.scr
 
 Target URL.
 
-## js-python-fetch-option-params
-* langs: js, python
+## js-fetch-option-params
+* langs: js
+- `params` <[Object]<[string], [string]|[number]|[boolean]>|[URLSearchParams]|[string]>
+
+Query parameters to be sent with the URL.
```

Comment:
Let's add examples to the documentation for the new parameter types, especially for the string one.